packages/hash/frontend/src/pages/signup.page.tsx

```diff
@@ -39,17 +38,18 @@ import {
   Action,
   InvitationInfo,
   parseGraphQLError,
-} from "../components/auth/utils";
-import { AuthLayout } from "../components/layout/PageLayout/AuthLayout";
-import { useGetInvitationInfo } from "../components/hooks/useGetInvitationInfo";
+} from "./shared/auth-utils";
+import { AuthLayout } from "./shared/auth-layout";
+import { useGetInvitationInfo } from "./shared/use-get-invitation-info";
+import { getPlainLayout, NextPageWithLayout } from "../shared/layout";
 
 enum Screen {
-  Intro,
-  VerifyCode,
-  AccountSetup,
-  AccountUsage,
-  OrgCreate,
-  OrgInvite,
+  IntroScreen,
+  VerifyCodeScreen,
+  AccountSetupScreen,
+  AccountUsageScreen,
+  OrgCreateScreen,
+  OrgInviteScreen,
 }
```

Comment:
I did not know we were using enums 😬 It'd be nice to get rid of them later.

Reply:
I agree. Initially implemented without having knowledge of the shortcomings of enums. Will update in a follow up PR